PLUB/Sources/Views/Meeting/CreateMeeting/Component/CategoryHeaderView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  CategoryHeaderView.swift
+//  PLUB
+//
+//  Created by 김수빈 on 2023/01/30.
+//
+
+import UIKit
+
+import SnapKit
+import RxSwift
+
+final class CategoryHeaderView: UIView {
+  private let disposeBag = DisposeBag()
+  
+  private let titleLabel = UILabel().then {
+    $0.text = """
+    취미에 진심인 우리!
+    어떤 플러빙을 시작해 볼까요?
+    """
+    $0.numberOfLines = 2
+    $0.textColor = .black
+    $0.font = .h4
+  }
+  
+  private let countStackView = UIStackView().then {
+    $0.axis = .vertical
+    $0.spacing = 2
+    $0.alignment = .center
+  }
+  
+  private let polygonImageView = UIImageView().then {
+    $0.image = UIImage(named: "polygonWhite")
+  }
+  
+  private let limitLabel = PaddingLabel(withInsets: 2, 2, 8, 8).then {
+    $0.text = "5개까지 선택할 수 있어요!"
+    $0.textColor = .black
+    $0.font = .overLine
+    $0.clipsToBounds = true
+    $0.layer.cornerRadius = 10
+    $0.backgroundColor = .white
+  }
+  
+  private let countLabel = UILabel().then {
+    $0.text = "0개"
+    $0.textColor = .mediumGray
+    $0.font = .h3
+    $0.textAlignment = .center
+  }
+  
+  private let selectedLabel = UILabel().then {
+    $0.text = "선택됨"
+    $0.textColor = .mediumGray
+    $0.font = .caption2
+    $0.textAlignment = .center
+  }
+  
+  init() {
+    super.init(frame: .zero)
+    setupLayouts()
+    setupConstraints()
+    setupStyles()
+    bind()
+  }
+  
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+}
+
+private extension CategoryHeaderView {
```

Comment:
첨언하자면, private extension 내부의 함수는 fileprivate로 적용되는 걸로 알고있습니다. Swift 5.7에서는 고쳐졌는지 모르겠네요 ㅎㅎ;
([링크](https://forums.swift.org/t/function-in-private-extension-must-be-declared-private-because-it-uses-a-private-type/44463))

Reply:
헛...저는 당연히 private일줄알았는데..지금까지 잘못알고있었네요..! fileprivate이면 저렇게 선언한 다른부분들도 일괄 수정해놓겠습니다!

Reply:
앗 수정하실 필요는 없습니다! 어차피 `fileprivate`여서 상관 없기는 해요. 저도 예전에 작업하다가 알게 된 사실이어서 공유하고자 말씀드린 거였습니다.

Reply:
헛 이미 수정해버렸습니다 ㅎㅎㅎㅎ알려주셔서 감사해요!!!